server/src/main/java/info/doseamigos/db/MySQLConnection.java

```diff
@@ -15,8 +15,8 @@
      */
     public static Connection create() throws SQLException {
         return DriverManager.getConnection(
-            "jdbc:mysql://doseamigos2.cdjwskv4xmf8.us-east-1.rds.amazonaws.com:3306/doseamigos2?&useLegacyDatetimeCode=false&serverTimezone=UTC",
-            "amigoadmin",
+            "jdbc:mysql://104.196.57.95:3306/doseamigos?&useLegacyDatetimeCode=false&serverTimezone=UTC",
```

Comment:
This points to the Google Cloud MySQL instance, which I'm using a free trial for the next 2 months.  It's security rules actually work, unlike Amazon RDS.